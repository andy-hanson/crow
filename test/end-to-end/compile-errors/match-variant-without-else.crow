main void()
	info log match x
	as x
		"hello"

v variant
x variant-member(v)
