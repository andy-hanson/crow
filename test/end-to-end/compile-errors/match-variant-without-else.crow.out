exit code: 0
stdout:
info: hello
stderr:
test/end-to-end/compile-errors/match-variant-without-else.crow 2:14-2:19 Match on a 'variant' must have an explicit 'else'.
